ext/pdo_pgsql/pgsql_driver.c

```diff
@@ -390,8 +403,11 @@ static int pdo_pgsql_get_attribute(pdo_dbh_t *dbh, zend_long attr, zval *return_
 			ZVAL_BOOL(return_value, H->disable_prepares);
 			break;
 
-		case PDO_ATTR_CLIENT_VERSION:
-			ZVAL_STRING(return_value, PG_VERSION);
+		case PDO_ATTR_CLIENT_VERSION: {
+			char buf[16];
+			pdo_libpq_version(buf, sizeof(buf));
+			ZVAL_STRING(return_value, buf);
+		}
 			break;
```

Comment:
Please move this break into the code block.

Reply:
Makes sense. Thanks!